prog/vm/github_runner.rb

```diff
@@ -126,12 +126,25 @@ def before_run
   end
 
   label def start
+    hop_waiting_for_customer_concurrency_limit unless concurrency_available?
+
     picked_vm = pick_vm
     github_runner.update(vm_id: picked_vm.id)
     picked_vm.update(name: github_runner.ubid.to_s)
+
     hop_wait_vm
   end
 
+  label def waiting_for_customer_concurrency_limit
+    Clog.emit("Waiting for customer capacity") { {github_runner: github_runner.values} }
+
+    if concurrency_available?
+      hop_start
+    else
+      nap 5
+    end
```

Comment:
No need to else part
```suggestion
    end
    nap 5
```